core/app/models/spree/address.rb

```diff
@@ -30,6 +30,11 @@ class Address < Spree::Base
     end
 
     validate :state_validate, :postal_code_validate
+    validates_uniqueness_of :location_name,
```

Comment:
`validates_uniqueness_of` helper is deprecated, we should use `validates :location_name, uniqueness: {`

Reply:
And this change also